server/desktop/src/main/java/dev/slimevr/desktop/tracking/trackers/hid/TrackersHID.kt

```diff
@@ -1,18 +1,15 @@
 package dev.slimevr.desktop.tracking.trackers.hid
 
-import com.jme3.math.FastMath
 import dev.slimevr.VRServer
 import dev.slimevr.tracking.trackers.Device
 import dev.slimevr.tracking.trackers.Tracker
 import dev.slimevr.tracking.trackers.TrackerStatus
-import dev.slimevr.tracking.trackers.udp.BoardType
-import dev.slimevr.tracking.trackers.udp.IMUType
-import dev.slimevr.tracking.trackers.udp.MCUType
-import dev.slimevr.tracking.trackers.udp.MagnetometerStatus
+import dev.slimevr.tracking.trackers.hid.HIDCommon
```

Comment:
Maybe also rename the class to DesktopHIDManager

Reply:
Actually maybe "Manager" *is* better since service usually implies it is providing access to something, whereas this is just accessing things.